pkgs/tools/package-management/poetry/unwrapped.nix

```diff
@@ -59,8 +59,8 @@ buildPythonPackage rec {
   ];
 
   pythonRelaxDeps = [
-    # only pinned to avoid dependency on Rust
-    "jsonschema"
+    # platformdirs 4.x is backwards compatible; https://github.com/python-poetry/poetry/commit/eb80d10846f7336b0b2a66ce2964e72dffee9a1c
```

Comment:
Why not use `fetchpatch`?

Reply:
Because there is nothing meaningful to fetch.

Reply:
i.e. that commit doesn't apply cleanly?

Reply:
To clarify, you want me to fetch a FOD, so that it fails to apply on the next bump, so we remember to remove it again?

Reply:
> To clarify, you want me to fetch a patch, so that it fails to apply on the next bump, so we remember to remove it again?

We don't update poetry in the batch upgrades, so we won't forget to remove the patch.
Anyway, if you prefer `pythonRelaxDeps` that's fine with me though I can't guarantee I won't forget to remove it.

Reply:
Poetry can break in python-updates, because it is a leaf-package, and we don't pay much attention to those. I have a slight preference for not pulling another FOD for this minor issue, since the hook has that exact use case.